modules/core/src/main/java/org/apache/ignite/internal/processors/service/LazyServiceConfiguration.java

```diff
@@ -40,6 +40,10 @@
     /** */
     private byte[] srvcBytes;
 
+    /** Known service methods to build service statistics. */
+    @GridToStringExclude
+    private String[] knownMtdNames;
```

Comment:
It used only for platform services and then this field is very misleading. I propose to store those methods inside of `PlatformService`. Is it possible?

Reply:
PlatformService, like any service, has many instances. ServiceCOnfig or service context has single instance. It's better to store one instance of shared resource/config.